data/categories.txt

```diff
@@ -346,7 +346,7 @@ cs:2Bar|pivo|pití|jídlo|voda|pitná voda|tekutiny
 sk:2Bar|pivo|pitie|jedlo|voda|pitná voda|tekutiny
 nl:2Bar|2pub|2café|bier|drinken|eten
 zh-Hant:1酒吧|酒館|飲酒|PUB|吧台|飲食
-pl:2Bar|2pub|piwo|napoje|jedzenie
+pl:2Bar|alkohole|drinki|napoje|jedzenie
```

Comment:
I've done this to make difference between bar and pub (place where you especially go to drink beer). We have two separate tags for these two places so I think It's good idea not to mix it. But you're right about "piwo" - it should stay with bar too.